src/nci_adapter.c

```diff
@@ -291,6 +291,9 @@ nci_adapter_convert_poll_b(
     dest->fsc = src->fsc;
     dest->nfcid0.bytes = src->nfcid0;
     dest->nfcid0.size = sizeof(src->nfcid0);
+    dest->prot_info.bytes = src->prot_info.bytes;
+    dest->prot_info.size = src->prot_info.size;
```

Comment:
If both prot_info's are `GUtilData` (AFAICT they both are), then just `dest->prot_info = src->prot_info;` would suffice.

Reply:
Thanks for the review, I have fixed that